test/system/comment_test.rb

```diff
@@ -359,6 +363,8 @@ def get_path(page_type, path)
       page.find('#text-input-edit-' + edit_id_num)
         .click
         .fill_in with: comment_text_edit
+        # taking into account the time for debounce function
+        sleep(0.7)
```

Comment:
@waridrox, what is the purpose of the sleep? Will the tests fail if the sleep is removed? And why 0.7 seconds?

Reply:
@RuthNjeri, here the purpose of `sleep` is to wait the tests for 0.7 seconds in order for the save function to take effect due to the debounce function, whenever new content is put in the comment text area: 
```javascript
    }, 700)
     );
   }  
     save(thisEditor) {
```

The tests would fail because this would be a new addition to the comment area hence the use of `sleep` is to update the tests.